scripts/review_weblog.py

```
@@ -82,41 +79,40 @@ def main():
-                    ],
-                    check=True,
-                )
-            except subprocess.CalledProcessError:
-                continue
+    counts = build_counts()
+
+    for ip, hits in counts.items():
+        if not should_block(ip, hits):
+            continue
+        try:
+            subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)